modules/core/src/v2/coins/xlm.ts

```diff
@@ -1,15 +1,17 @@
 import * as _ from 'lodash';
-const BigNumber = require('bignumber.js');
+import { BigNumber } from 'bignumber.js';
 import * as querystring from 'querystring';
 import * as url from 'url';
 import * as Bluebird from 'bluebird';
+import * as bitcoin from 'bitgo-utxo-lib';
 const co = Bluebird.coroutine;
-const request = require('superagent');
+import * as request from 'superagent';
+import * as stellar from 'stellar-sdk';
```

Comment:
Is `import *` fairly common TS pattern? I've always found it more readable + maintainable to be explicit with imports.

Reply:
It's the closest analogue to the old pattern of `const lib = require('some-lib')`. I could pull out just the members I need, but that would mean more code changes which are not related to the fix. I'm trying to improve things a little bit at a time instead of all at once - it's easy to get sucked into a rabbit hole if you try to fix too much.